src/Notification.tsx

```diff
@@ -0,0 +1,51 @@
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+
+type Props = {
+  errorFromServer: boolean,
+  setErrorFromServer: (errorFromServer: boolean) => void,
+  errorMessage: string,
+};
+
+export const Notifications: React.FC<Props> = ({
+  errorFromServer,
+  setErrorFromServer,
+  errorMessage,
+}) => {
+  const [isHidden, setisHidden] = useState(true);
+
+  useEffect(() => {
+    const notificationCloser = () => {
+      const timerId = window.setTimeout(() => {
+        setisHidden(true);
+        setErrorFromServer(false);
+      }, 3000);
+
+      window.clearTimeout(timerId);
+    };
+
+    if (errorFromServer) {
```

Comment:
you can use the `errorMessage`
```suggestion
    if (errorMessage.length > 0) {
```